app/src/ui/delete-branch/delete-branch-dialog.tsx

```diff
@@ -61,7 +98,7 @@ export class DeleteBranch extends React.Component<
   }
 
   private renderDeleteOnRemote() {
-    if (this.props.branch.remote && this.props.existsOnRemote) {
+    if (this.props.branch.remote && this.state.existsOnRemote === true) {
```

Comment:
No need for the `... === true`, `this.state.existsOnRemote` should do just fine.